dask/dataframe/core.py

```diff
@@ -1523,6 +1523,14 @@ def query(self, expr, **kwargs):
                                  dummy, self.divisions)
 
     @derived_from(pd.DataFrame)
+    def eval(self, expr, inplace=None, **kwargs):
+        if '=' in expr and inplace:
```

Comment:
I think I need to check that `inplace is not False`.  See recent changes and test